libs/core/src/components/sage-divider/sage-divider.tsx

```diff
@@ -0,0 +1,44 @@
+import { Component, Prop, Host, h } from '@stencil/core';
+
+@Component({
+  tag: 'sage-divider',
+  styleUrl: 'sage-divider.scss',
+  shadow: true,
+})
+export class SageDivider {
+  /**
+   * Sets divider to display vertically
+   * @defaultValue false
+   */
+  @Prop() vertical = false;
+
+  /**
+   *
+   * Adds offset margin/padding to expand the width (horizontal) or the height (vertical) of divider.
+   */
+  @Prop() offset: 'xxs' | 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl';
+
+  private classNames() {
```

Comment:
We'll discuss this in more detail in the future, but I wonder if this should evolve into a global utility function.